src/Analyzer.Cli/Loggers/ConsoleLoggerFormatter.cs

```diff
@@ -11,24 +12,35 @@ namespace Microsoft.Azure.Templates.Analyzer.Cli
     /// </summary>
     public sealed class ConsoleLoggerFormatter : ConsoleFormatter
     {
-        const string DefaultForegroundColor = "\x1B[39m\x1B[22m";
+        private const string defaultForegroundColor = "\x1B[39m\x1B[22m";
```

Comment:
```suggestion
        private const string DefaultForegroundColor = "\x1B[39m\x1B[22m";
```

Reply:
We use lowercase for private variables

Reply:
I know we don't have it written down in the style guidelines, but I usually use capital first letter for constants to differentiate them from variables. (By definition, a constant isn't a variable.)

Reply:
In [here](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers), under field in the table, they put `const` as Pascal, but it is also `public`. I am fine with either way